### DISCUSSION TITLE
Dashboard visualization

### DISCUSSION THREAD

**Discussion by anhluu-nguyen** [langgenius/dify#4490] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

I am trying to have the AI gather information as the conversation between user and chatbot progresses, and to create a dashboard of important information.

### 2. Describe the feature you'd like to see

A dashboard generator based off a conversation between user and chatbot

### 3. How will this feature improve your workflow or experience?

It will provide a better user experience and would be beneficial for the company I am working for.

### 5. Can you help us with this feature?

- [X] I am interested in contributing to this feature.

**Comment by ZuzooVn** [comment]
+1

**Comment by loganclark360** [comment]
I would like the coding  set on easy then I can switch it once I am more
comfortable.
Thanks you

W. Logan Clark

**Reply by anhluu-nguyen** [reply]
What do you mean by that?

**Reply by loganclark360** [reply]
I mean. I would like it to be set on the easiest code to learn and adapt to.then once  I feel comfortable, I will let you know and we can make t
We ca We can make changes.

**Reply by anhluu-nguyen** [reply]
Sorry I am a little lost would you like me to do something or help?
